Bio/SeqIO/GfaIO.py

```diff
@@ -0,0 +1,177 @@
+"""Bio.SeqIO support for the Graphical Fragment Assembly format.
+
+This format is output by many assemblers and includes linkage information for
+how the different sequences fit together, however, we just care about the
+segment (sequence) information.
+
+Documentation:
+    Version 1.x: https://gfa-spec.github.io/GFA-spec/GFA1.html
+    Version 2.0: https://gfa-spec.github.io/GFA-spec/GFA2.html
```

Comment:
This does not render nicely as RST, maybe make it into a bullet point list explicit?